docker.gradle

```diff
@@ -92,7 +92,7 @@ task copyContainerFiles(type: Copy) {
 task buildImage(type: DockerBuildImage) {
   dependsOn copyContainerFiles
   inputDir = file(build_context)
-  tag = System.getenv('DOCKER_REPO_URL') + '/uaa:' + version
+  image = System.getenv('DOCKER_REPO_URL') + '/uaa:' + version
```

Comment:
This commit could be squashed with the previous one that is the cause for it being needed.